Apeswap-Bots/token-fee-update/src/agent.spec.ts

```diff
@@ -0,0 +1,128 @@
+import {
+  Finding,
+  FindingType,
+  FindingSeverity,
+  HandleTransaction,
+  ethers,
+} from "forta-agent";
+import {
+  createAddress,
+  MockEthersProvider,
+  MockEthersSigner,
+  TestTransactionEvent,
+} from "forta-agent-tools/lib/tests";
+import { handleTransaction } from "./agent";
+import utils from "./utils";
+
+const TEST_REFLECT_TOKEN: string = createAddress("0xcdcd");
+
+const previousFee = "1";
+const currentFee = "2";
+
+const findingTestCases = [
+  Finding.fromObject({
+    name: "Detect Fees Related To The Token",
+    description: `Token tax fee has been changed`,
+    alertId: "APESWAP-3",
+    severity: FindingSeverity.Info,
+    type: FindingType.Info,
+    protocol: "Apeswap",
+    metadata: {
+      previousFee: previousFee,
+      currentFee: currentFee,
+    },
+    addresses: [],
+  }),
+];
+
+const updateTaxFee = async (
```

Comment:
The `mockSigner` and this whole function can be replaced by just the `TestTransactionEvent().addAnonymousEventLog(..)`